index.html

```diff
@@ -1471,7 +1472,7 @@ <h2>Key Agreement</h2>
 
             <p>
 The `keyAgreement` [=verification relationship=] is used to
-specify how an entity can generate encryption material in order to transmit
+specify how an entity can generate a content encryption key in order to transmit
```

Comment:
I think this is a little too specific and might lead an implementer to think that they can't use key encapsulation or that some other hairy / overly prescriptive things might block them from using this for encryption when that's not intent here. I recommend this more general text:

```suggestion
specify how an entity can perform encryption in order to transmit
```